docs/framework/wcf/simplified-configuration.md

```diff
@@ -6,84 +6,84 @@ ms.topic: reference
-  
-```xml  
-<system.serviceModel>  
-  <behaviors>  
-    <serviceBehaviors>  
-      <behavior name="MyServiceBehavior">  
-        <serviceMetadata httpGetEnabled="true" />  
-        <serviceDebug includeExceptionDetailInFaults="false" />  
-      </behavior>  
-    </serviceBehaviors>  
-  </behaviors>  
-  <bindings>  
-   <basicHttpBinding>  
-      <binding name=MyBindingConfig"  
-               maxBufferSize="100"  
-               maxReceiveBufferSize="100" />  
-   </basicHttpBinding>  
-   </bindings>   <services>  
-    <service behaviorConfiguration="MyServiceBehavior"  
-             name="MyService">  
-      <endpoint address=""  
-                binding="basicHttpBinding"  
-                contract="ICalculator"  
-                bindingConfiguration="MyBindingConfig" />  
-      <endpoint address="mex"  
-                binding="mexHttpBinding"  
-                contract="IMetadataExchange"/>  
-    </service>  
-  </services>  
-</system.serviceModel>  
-```  
-  
- [!INCLUDE[netfx_current_short](../../../includes/netfx-current-short-md.md)] makes configuring a WCF service easier by removing the requirement for the <`service`> element. If you do not add a <`service`>  section or add any endpoints in a <`service`> section and your service does not programmatically define any endpoints, then a set of default endpoints are automatically added to your service, one for each service base address and for each contract implemented by your service. In each of these endpoints, the endpoint address corresponds to the base address, the binding is determined by the base address scheme and the contract is the one implemented by your service. If you do not need to specify any endpoints or service behaviors or make any binding setting changes, you do not need to specify a service configuration file at all. If a service implements two contracts and the host enables both HTTP and TCP transports the service host creates four default endpoints, one for each contract using each transport. To create default endpoints the service host must know what bindings to use. These settings are specified in a <`protocolMappings`> section within the <`system.serviceModel`> section. The <`protocolMappings`> section contains a list of transport protocol schemes mapped to binding types. The service host uses the base addresses passed to it to determine which binding to use. The following example uses the <`protocolMappings`> element.  
-  
+Configuring Windows Communication Foundation (WCF) services can be a complex task. There are many different options and it is not always easy to determine what settings are required. While configuration files increase the flexibility of WCF services, they also are the source for many hard to find problems. .NET Framework 4.6.1 addresses these problems and provides a way to reduce the size and complexity of service configuration.
```

Comment:
Wouldn't netfx-current be 4.8 actually, so if we replace it with a specific older version should we add "and later versions" (for netfx-current-short and netfx-current-long)?

Reply:
True, those replacements will require a little more thought other than blanket find/replace, so I've left them unchanged for now (reverted my changes).